test/vinyl/gh-5823-skip-newer-than-snap-vylog.result

```diff
@@ -0,0 +1,81 @@
+-- test-run result file version 2
+test_run = require('test_run').new()
+ | ---
+ | ...
+
+-- Test is about following scenario:
+-- 1. There's both memtx and vinyl data;
+-- 2. User starts checkpoint process;
+-- 3. In the most unsuitable moment instance crashes;
+-- 4. Recovering in the casual mode does not help;
+-- 5. Recovering in the force recovery mode solves the problem (deletes
+--    redundant vylog file).
+--
+test_run:cmd("create server test with script='vinyl/gh-5823-crash_snapshot.lua'")
+ | ---
+ | - true
+ | ...
+test_run:cmd("start server test with args='2' with crash_expected=True")
+ | ---
+ | - false
+ | ...
+-- Can't bootstrap instance without force_recovery.
+--
+test_run:cmd("start server test with args='0' with crash_expected=True")
+ | ---
+ | - false
+ | ...
+
+fio = require('fio')
```

Comment:
AFAIS, `test_run:grep_log()` allows to pass a file name option to do the same.

Reply:
Ok, but anyway it is removed in the last commit